src/plugins/github/depends/__init__.py

```diff
@@ -35,6 +36,18 @@ def get_labels(event: PullRequestEvent | IssuesEvent):
     return labels
 
 
+def get_name_by_labels(labels: LabelsItems = Depends(get_labels)) -> list[str]:
```

Comment:
这个不应该叫 `get_labels_name` 吗（
```suggestion
def get_labels_name(labels: LabelsItems = Depends(get_labels)) -> list[str]:
```